src/extensions/cli/cli.js

```diff
@@ -10,182 +10,211 @@ const { spawn } = require('child_process')
 const path = require('path')
 
 /**
- * The Current Working Directory.
+ * Cli extension.
  *
- * @type {string}
+ * @class
  */
-let cwd
+class Cli {
+    constructor() {
+        /**
+         * The Current Working Directory.
+         *
+         * @type {string}
+         */
+        this.cwd = process.cwd()
+
+        /**
+         * An object containing environment variables to inject when running your command.
+         *
+         * @type {Object}
+         */
+        this.env = {}
+
+        /** @type {string} */
+        this.killSignal = null
+
+        /** @type {number} */
+        this.killDelay = 0
+
+        /**
+         * Latest command execution exit code.
+         *
+         * @type {number}
+         */
+        this.exitCode = null
+
+        /**
+         * The command's output.
+         *
+         * @type {string}
+         */
+        this.stdout = ''
+
+        /**
+         * The command's error output.
+         *
+         * @type {string}
+         */
+        this.stderr = ''
+    }
 
-/**
- * An object containing environment variables to inject when running your command.
- *
- * @type {Object}
- */
-let env = {}
+    /**
+     * Sets the Current Working Directory for the command.
+     *
+     * @param {string} _cwd - The new CWD
+     */
+    setCwd(_cwd) {
+        if (_cwd.indexOf('/') === 0) {
+            this.cwd = _cwd
+        } else {
+            this.cwd = path.join(process.cwd(), _cwd)
+        }
+    }
 
-/** @type {string} */
-let killSignal
+    /**
+     * Returns Current Working Directory.
+     *
+     * @return {string}
+     */
+    getCwd() {
+        return this.cwd
+    }
 
-/** @type {number} */
-let killDelay
+    /**
+     * Defines environment variables.
+     * Beware that all existing ones will be overridden!
+     *
+     * @param {Object} _env - The environment variables object
+     */
+    setEnvironmentVariables(_env) {
```

Comment:
You can remove the `_`, it was only there to not conflict with var name which is now bound to `this`